src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenRefLocalTests.cs

```diff
@@ -1697,5 +1698,31 @@ static void M()
             Assert.Null(model.GetSymbolInfo(refInt).Symbol);
             Assert.Null(model.GetTypeInfo(refInt).Type);
         }
+
+        [WorkItem(17395, "https://github.com/dotnet/roslyn/issues/17453")]
+        [Fact]
+        public void Regression17395()
+        {
+            var source = @"
+using System;
+
+public class C 
+{            
+    public void F()
+    {
+        ref int[] a = ref {1,2,3};
+        Console.WriteLine(a[0]);
+    }        
+}
+";
+
+            var c = CreateCompilationWithMscorlib45AndCSruntime(source);
```

Comment:
CreateCompilationWithMscorlib should be sufficient.

---
In reply to: [103611013](https://github.com/dotnet/roslyn/pull/17477#discussion_r103611013) [](ancestors = 103611013)